WD_GWB.yml

```diff
@@ -2,134 +2,11 @@ name: WD_GWB
 channels:
   - defaults
 dependencies:
-  - _libgcc_mutex=0.1
-  - _openmp_mutex=5.1
-  - astropy=6.1.0
-  - astropy-iers-data=0.2024.6.3.0.31.14
-  - asttokens=2.0.5
-  - blas=1.0
-  - bottleneck=1.3.7
-  - brotli=1.0.9
-  - brotli-bin=1.0.9
-  - bzip2=1.0.8
-  - ca-certificates=2024.7.2
-  - comm=0.2.1
-  - contourpy=1.2.0
-  - cycler=0.11.0
-  - cyrus-sasl=2.1.28
-  - dbus=1.13.18
-  - debugpy=1.6.7
-  - decorator=5.1.1
-  - executing=0.8.3
-  - expat=2.6.2
-  - fontconfig=2.14.1
-  - fonttools=4.51.0
-  - freetype=2.12.1
-  - glib=2.78.4
-  - glib-tools=2.78.4
-  - gst-plugins-base=1.14.1
-  - gstreamer=1.14.1
-  - icu=73.1
-  - intel-openmp=2023.1.0
-  - ipykernel=6.28.0
-  - ipython=8.25.0
-  - jedi=0.19.1
-  - jpeg=9e
-  - jupyter_client=8.6.0
-  - jupyter_core=5.7.2
-  - kiwisolver=1.4.4
-  - krb5=1.20.1
-  - lcms2=2.12
-  - ld_impl_linux-64=2.38
-  - lerc=3.0
-  - libbrotlicommon=1.0.9
-  - libbrotlidec=1.0.9
-  - libbrotlienc=1.0.9
-  - libclang=14.0.6
-  - libclang13=14.0.6
-  - libcups=2.4.2
-  - libdeflate=1.17
-  - libedit=3.1.20230828
-  - libffi=3.4.4
-  - libgcc-ng=11.2.0
-  - libgfortran-ng=11.2.0
-  - libgfortran5=11.2.0
-  - libglib=2.78.4
-  - libgomp=11.2.0
-  - libiconv=1.16
-  - libllvm14=14.0.6
-  - libpng=1.6.39
-  - libpq=12.17
-  - libsodium=1.0.18
-  - libstdcxx-ng=11.2.0
-  - libtiff=4.5.1
-  - libuuid=1.41.5
-  - libwebp-base=1.3.2
-  - libxcb=1.15
-  - libxkbcommon=1.0.1
-  - libxml2=2.10.4
-  - lz4-c=1.9.4
-  - matplotlib=3.8.4
-  - matplotlib-base=3.8.4
-  - matplotlib-inline=0.1.6
-  - mkl=2023.1.0
-  - mkl-service=2.4.0
-  - mkl_fft=1.3.8
-  - mkl_random=1.2.4
-  - mysql=5.7.24
-  - ncurses=6.4
-  - nest-asyncio=1.6.0
-  - numexpr=2.8.7
-  - numpy=1.26.4
-  - numpy-base=1.26.4
-  - openjpeg=2.4.0
-  - openssl=3.0.14
-  - packaging=24.1
-  - pandas=2.2.2
-  - parso=0.8.3
-  - pcre2=10.42
-  - pexpect=4.8.0
-  - pillow=10.4.0
-  - pip=24.0
-  - platformdirs=3.10.0
-  - ply=3.11
-  - prompt-toolkit=3.0.43
-  - prompt_toolkit=3.0.43
-  - psutil=5.9.0
-  - ptyprocess=0.7.0
-  - pure_eval=0.2.2
-  - pybind11-abi=5
-  - pyerfa=2.0.1.4
-  - pygments=2.15.1
-  - pyparsing=3.0.9
-  - pyqt=5.15.10
-  - pyqt5-sip=12.13.0
-  - python=3.12.4
-  - python-dateutil=2.9.0post0
-  - python-tzdata=2023.3
-  - pytz=2024.1
-  - pyyaml=6.0.1
-  - pyzmq=25.1.2
-  - qt-main=5.15.2
-  - readline=8.2
-  - scipy=1.13.1
-  - seaborn=0.13.2
-  - setuptools=69.5.1
-  - sip=6.7.12
-  - six=1.16.0
-  - sqlite=3.45.3
-  - stack_data=0.2.0
-  - tbb=2021.8.0
-  - tk=8.6.14
-  - tornado=6.4.1
-  - traitlets=5.14.3
-  - tzdata=2024a
-  - unicodedata2=15.1.0
-  - wcwidth=0.2.5
-  - wheel=0.43.0
-  - xz=5.4.6
-  - yaml=0.2.5
-  - zeromq=4.3.5
-  - zlib=1.2.13
-  - zstd=1.5.5
```

Comment:
The following packages could not be found:
  - libcups=2.4.2
  - libedit=3.1.20230828
  - mysql=5.7.24
  - libxkbcommon=1.0.1
  - gstreamer=1.14.1
  - libxcb=1.15
  - _openmp_mutex=5.1
  - libuuid=1.41.5
  - ld_impl_linux-64=2.38
  - libgfortran-ng=11.2.0
  - gst-plugins-base=1.14.1
  - readline=8.2
  - libgomp=11.2.0
  - ncurses=6.4
  - dbus=1.13.18
  - libgfortran5=11.2.0
  - libgcc-ng=11.2.0
  - libstdcxx-ng=11.2.0